companion-select-text-insert/info.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "Insert selected text from web companion addon and Append current URL below current line ",
+  "identifier": "companion-select-text-insert",
+  "script": "selected-text-below.qml",
+  "authors": ["@v3ss0n"],
+  "platforms": ["linux", "macos", "windows"],
+  "version": "0.1.0",
+  "minAppVersion": "18.04.3",
+  "description" : "This script insert selected text from webpage and append the URL below current line.Requires  Web Companion addon for Browsers : https://github.com/qownnotes/web-companion/"
```

Comment:
s/insert/inserts/
s/append/appends/
s/line.Requires  Web Companion addon for Browsers :/line. Requires the Web Companion browser extension:/